lib/methods/save.js

```diff
@@ -9,6 +9,7 @@ var _ = require('lodash')
  */
 exports.save = function (Model, instance, callback) {
   var table = this.getTableName(Model)
+  table = this.escapeKeys([table])[0]
   var payload = instance.toPayload()
   var primaryKeyColumn = this.getPrimaryKeyColumn(Model)
   var columns = this.escapeKeys(this.fetchColumns(table, payload))
```

Comment:
because of this: `var columns = this.escapeKeys(this.fetchColumns(table, payload))`